examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -30,144 +25,237 @@ func Render(_ string) string {
-- [Get testnet GNOTs](https://faucet.gno.land)
+## Key Features of Gno
 
-## Explore the universe
+- Open & Transparent: All applications are open-source and fully auditable.
+- Composable: Smart contracts are modular, reusable, and customizable to ease development and extend the ecosystem’s capabilities quickly.
+- Censorship-Resistant: Enables platforms with fair moderation and freedom of communication without a central authority.
+- Developer-Friendly: Intuitive design that reduces onboarding timelines and enables faster iteration.
+- Go-Based & Type Safe: Go developers can transition seamlessly to Gno thanks to the well-established Go language, while benefiting from type safety for security.
 
-- [Discover demo packages](https://github.com/gnolang/gno/tree/master/examples)
-- [Gnoscan](https://gnoscan.io)
-- [Portal Loop](https://docs.gno.land/concepts/portal-loop)
-- [Testnet 4](https://test4.gno.land/)
-- [Faucet Hub](https://faucet.gno.land)
 
-## [Latest Blogposts](/r/gnoland/blog)
+---
 
-:latest-blogposts:
+Use Cases
 
-## [Latest Events](/r/gnoland/events)
+## Why Gno and gno.land?
 
-:upcoming-events:
+gno.land’s smart contract platform is a first step in an experiment to build a complete dOS to bypass the centralized and controlled restrictions of the current Web, and power new social platforms, governance frameworks, and economic models that favor contribution-based incentives.
+
+<gno-columns>
+
+####
+
+![The next killer app](#)
+
+### A Platform Designed for Next Generation Apps
+
+A reusable technical architecture for everyone to build the next generation applications of the future that attract the next wave of builders and users.
+
+####
+
+![New-Economic-Model](#)
+
+### New Economic Model
+
+A builder-aligned economy that incentivizes developers, content creators, and maintainers with sustainable rewards.
+
+####
+
+![Decentralized Platform Governance](#)
+
+### Decentralized Governance: GovDAO
+
+A governance model that prioritizes contribution-based participation over capital-weighted control.
+
+####
+
+![Social coordination & community](#)
 
-## [Hall of Fame](/r/leon/hof)
+### Social coordination & community
 
-:latest-hof:
+A decentralized space for uncensored discourse, fair moderation, and community-driven governance with forkability to allow for conflict resolution.
+
+
+
+
+</gno-columns>
 
 ---
+Beta Mainnet
+
+## Beta Mainnet Coming Soon!
+
+Stay tuned for the first phase of gno.land's launch, the beta mainnet release.  
+- [Road to Mainnet and Beyond](https://gno.land/r/gnoland/blog:p/road-to-mainnet)
+
+---
+
+Ecosystem
+
+## Discover the gno.land universe
+
+Explore gno.land's expanding ecosystem of applications, wallets, and explorers.
+
+[/the ecosystem ](https://gno.land/ecosystem)
+
+
+<gno-columns>
+
+####
+
+![Wallet](#)
+
+### Wallets and Faucet
+
+Execute transactions on gno.land using wallets, or a CLI keychain and client tool.
```

Comment:
Leon, I think we have better content on docs.gno.land. Let's keep the homepage light while inviting visitors to go to docs.gno.land for more information about the tools.